src/net_processing.cpp

```diff
@@ -4117,7 +4117,7 @@ void PeerLogicValidation::ProcessMessage(
         }
 #endif // ENABLE_WALLET
         coinJoinServer->ProcessMessage(pfrom, msg_type, vRecv);
-        sporkManager->ProcessSporkMessages(pfrom, msg_type, vRecv, m_connman);
+        sporkManager->ProcessMessage(pfrom, msg_type, vRecv, m_connman);
```

Comment:
tbh I see no benefit in this commit... I'd probably prefer dropping it...

Reply:
It was for the sake of consistency, every other manager has a ProcessMessage except for SporkManager so that was corrected.